boards/arm/particle_xenon/doc/particle_xenon.rst

```diff
@@ -0,0 +1,456 @@
+.. _reel_board:
```

Comment:
Hmmm... Looks like you copied the reel board documentation (as a starting place) but then didn't edit it to become the particle xenon board documentation.  Oops. :)

Reply:
It's todo ;) Just needed a quick board up for a friend. I'll fix tomorrow